StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -139,6 +139,22 @@ final class LinkAccountPickerViewController: UIViewController {
                 self.dataSource
                     .analyticsClient
                     .logMerchantDataAccessLearnMore(pane: .linkAccountPicker)
```

Comment:
> just trying to confirm nothing else would be displaying the notice

yea nothing else is displaying the notice

you are spot on, however, that this is a bit confusing _(P.S. you really are good at catching various small intricacies!)_ the TL;DR is that the code behind this used to present SafariVC browser (that's what it does on `master` branch), but now we want to present Native UI. Abstracting presenting SafariVC was easy because it only needs a URL (which the underlying code knows). Abstracting presenting Native UI is not as easy because we need to feed custom data into Native UI (which the underlying code does not know about).